prebid-mobile/prebid-mobile.md

```diff
@@ -11,31 +11,93 @@ nav_section: prebid-mobile
- 2. Our Prebid Mobile module sends this bid information to your primary ad server SDK using key/value targeting.
- 3. The primary ad server SDK sends a request to the primary ad server including these custom parameters.  These parameters will match the key/value targeting previously configured on Prebid line items by your ad ops teams.
- 4. If the Prebid Mobile bid wins, the primary ad server returns our creative JS to your primary ad server SDK's ad view.
- 5. The creative JS in the ad view fetches the actual cached creative content from Prebid Server. When this happens, the impression is counted.
+Prebid Mobile works in conjunction with Prebid Server, demand partners, and your primary ad server to deliver programmable demand to your mobile app.
+
+### What You Need to Get Started
+
+Before you begin to use Prebid Mobile in your apps, you need to prepare your end-to-end system.
+
+**Mobile Developers**
+
+The following is a one-time setup process for *mobile developers*:
+
+-   [Set up a Prebid Server account]({{site.github.url}}/prebid-mobile/prebid-mobile-pbs.html).
+
+-   Download the Prebid Mobile SDK:
+    -   [SDK for iOS](https://github.com/prebid/prebid-mobile-ios)
+    -   [SDK for Android](https://github.com/prebid/prebid-mobile-android)
+
+After the initial setup, ad slots and line items will need to be set up and maintained:
+
+-   *Mobile developers* register ad units with the Prebid Mobile framework.
+    -   [iOS Code Integration]({{site.github.url}}/prebid-mobile/code-integration-ios.html)
+    -   [Android Code Integration]({{site.github.url}}/prebid-mobile/code-integration-android.html)
+
+**Ad Ops**
+
+-   *Ad ops team members* configure their primary ad server with Prebid Mobile line items targeted to key/values.
+    -   [Set Up Line Items for DFP]({{site.github.url}}/prebid-mobile/adops-line-item-setup-dfp.html)
+    -   [Set Up Line Items for MoPub]({{site.github.url}}/prebid-mobile/adops-line-item-setup-mopub.html)
+
+### Lifecycle of a Mobile Ad Call
+
+After you've set up Prebid Mobile and created your line items on your ad server, your ad calls go through the following workflow:
 
 {: .pb-img.pb-lg-img :}
-![How Prebid Mobile Works - Diagram]({{site.baseurl}}/assets/images/prebid-mobile/prebid-mobile.png)
+![How Prebid Mobile Works - Diagram]({{site.baseurl}}/assets/images/prebid-mobile/pbm-overview-flow.png)
+
+1.  Prebid Mobile calls Prebid Server, sending along ad unit information.
+2.  Prebid Server passes ad unit information on to the demand partners.
+3.  Each demand partner conducts an internal auction and sends the top bids to Prebid Server.
+4.  Prebid Server returns key/value data, including bid price, to the Prebid Mobile app.
+5.  Prebid Mobile sends this key/value data to the primary ad server. (This data will match the key/value targeting previously configured on Prebid line items by the ad ops team.)
```

Comment:
Not clear what it means that "this data will match the key/value targeting previously configured on the Prebid line items".

Reply:
My understanding was that the key/value data pbm receives with the bid is matched up against the LI targeting data configured on the ad server. Is that right?

Reply:
Prebid Mobile sends a request to Prebid Server.  Prebid Server sends requests to demand partners.  Demand partners respond to Prebid Server with bids (including price and creative content).  Prebid Server responds to Prebid Mobile with bids.  Prebid Mobile sets key-value targeting per-slot through the primary ad server SDK.  This key-value targeting will match key-value targeting set on preconfigured ad server LIs.  If the Prebid LI is selected by the primary ad server, then the primary ad server will return the Prebid Mobile creative JS into the primary ad server ad view.  The Prebid Mobile creative JS will fetch the corresponding creative content from the winning Prebid Server demand partner.